src/view/forms/SnoozeInputs.tsx

```diff
@@ -4,21 +4,73 @@ import UsaTextInput from "./UsaTextInput";
 import UsaNumberInput from "./UsaNumberInput";
 import AddNoteInput from "./AddNoteInput";
 
+const technicalSubtypes: TechnicalSubtype[] = [
+  {
+    value: "tecs_check",
+    text: "Cannot refresh TECS Check"
+  },
+  {
+    value: "undo_referral",
+    text: "Undo referral"
+  },
+  {
+    value: "unable_to_update_dob",
+    text: "Unable to Update DOB"
+  },
+  {
+    value: "unable_to_update_address",
+    text: "Unable to update address"
+  },
+  {
+    value: "unable_to_assign",
+    text: "Unable to assign/un-assign"
+  },
+  {
+    value: "card_production_error",
+    text: "Card Production Error"
+  },
+  {
+    value: "stuck_in_a_state",
+    text: "Case stuck in [x] state, and needs to be pushed to next phase"
```

Comment:
Maybe I missed it, is this actually displayed as `[x] state` or do we do a replacement somewhere?

Reply:
this is a good question. Raised it on the corresponding trello card

Reply:
In live discussion, suggested to change this to "current state".